src/tools/compiletest/src/main.rs

```diff
@@ -1,5 +1,6 @@
 #![crate_name = "compiletest"]
 #![feature(test)]
+#![feature(try_blocks)]
```

Comment:
Please avoid introducing unstable features to compiletest, as it's intended to be largely "works on stable." The other two features here already cause us some level of pain and we should avoid increasing that :)

Reply:
Removed.